src/sql/workbench/contrib/notebook/browser/calloutDialog/imageCalloutDialog.ts

```diff
@@ -186,7 +186,7 @@ export class ImageCalloutDialog extends CalloutDialog<IImageCalloutDialogOptions
 	public insert(): void {
 		this.hide();
 		this._selectionComplete.resolve({
-			insertMarkup: `<img src="${strings.escape(this._imageUrlInputBox.value)}">`,
+			insertMarkdown: `<img src="${strings.escape(this._imageUrlInputBox.value)}">`,
```

Comment:
It's really suspposed to have markdown, so it will look like `![]()`. I just haven't changed the imageCallout dialog yet. Just went ahead and changed Markup->Markdown in the notebook codebase